packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -311,7 +312,7 @@ class EnrollVerbHandler extends AbstractVerbHandler {
 
   /// Returns a Map where key is an enrollment key and value is a
   /// Map of "appName","deviceName" and "namespaces"
-  Future<String> _fetchEnrollmentRequests(
+  Future<String> _fetchEnrollmentRequests(EnrollParams? enrollVerbParams,
```

Comment:
instead of null enrollVerbParams..add enrollVerbParams as optional param

Reply:
This has been changed in [fix: make verbParams optional param from fetchEnrollments()](https://github.com/atsign-foundation/at_server/pull/1846/commits/ffebc36ffdb07cc4133207cbdb68c9063c6aa7e0)